firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/DataCollectionArbiter.java

```diff
@@ -142,7 +142,7 @@ private void logDataCollectionState(boolean dataCollectionEnabled) {
             ? "global Firebase setting"
             : setInManifest ? FIREBASE_CRASHLYTICS_COLLECTION_ENABLED + " manifest flag" : "API";
     Logger.getLogger()
-        .d(
+        .v(
```

Comment:
This is another one that this is pretty useful for troubleshooting so I'd prefer to keep it as DEBUG.

Reply:
👍